src/components/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a, b }) {
+  return (
+    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+  );
```

Comment:
I think content  and tag better write in different line

Reply:
It's not critical here